notifications_utils/clients/redis/redis_client.py

```diff
@@ -147,6 +147,14 @@ def incr(self, key, raise_exception=False):
             except Exception as e:
                 self.__handle_exception(e, raise_exception, "incr", key)
 
+    def incrby(self, key, by, raise_exception=False):
```

Comment:
I dont understand this name but I guess its the same naming convention as above

Reply:
Yeah, basically we're wrapping the redis [incrby](https://redis.io/commands/incrby/) function here, so I kept to the same "use the name of the redis function" that the rest are using.